packages/components/src/local/route/helpers/history-routes-for.ts

```diff
@@ -0,0 +1,38 @@
+import { LatLng } from 'leaflet'
+
+import SergeHex from '../../mapping/types/serge-hex'
+import SergeGrid from '../../mapping/types/serge-grid'
+import hexNamed from '../../hex-grid/helpers/hex-named'
+import { RouteData } from './planned-routes-for'
+import { LENGTH_OF_TRIMMED_LINE } from './planned-routes-for'
+
+const historyRoutesFor = (gridCells: SergeGrid<SergeHex<{}>>, position: string, steps: [any], 
+  trimmed: boolean): RouteData => {
+    const polyline: LatLng[] = []
+    const turnEnds: LatLng[] = []
+    var stepCtr = 0
+    // start with current position
+    const startCell: SergeHex<{}> | undefined = hexNamed(position, gridCells)
+    if(startCell) {
+      const startPos: LatLng = startCell.centreLatLng
+      if(steps) {
+        // store the line start
+        polyline.push(startPos)
+        steps.forEach((step:any) => {
```

Comment:
Is `any` the correct type here?

This also feels like it could be refactored in a more `es6` way can you flag it with a `//TODO:` and I'll go through these when I find the time.

Reply:
Good question @foxleigh81 .  It's not actually `any`, but it's a structure that we don't (yet) have defined in TS.

I'm happy to produce a TS definition for it, if you want me to.  But, _somehow_ I'll have to cast the mock data object to the definition, and I don't know how to do that.

Reply:
Nah, leave it for now :)